src/sap/fhir/model/r4/lib/FHIRRequestor.js

```diff
@@ -151,7 +151,12 @@ sap.ui.define([
 			sETag = oBindingInfo.getETag();
 		}
 		var oFHIRBundleRequest = new FHIRBundleRequest(oBinding, sMethod, sRequestUrl, fnSuccess, fnError, sETag);
-		var oFHIRBundleEntry = new FHIRBundleEntry(sFullUrl, oResource, oFHIRBundleRequest);
+		var oFHIRBundleEntry;
+		if (sMethod == HTTPMethod.POST || sMethod == HTTPMethod.PUT) {
```

Comment:
why do we need this if here? In which cases we ignore the `oResource`?

Reply:
during the DELETE or GET. I see it was sending the resource during the delete request